src/data/memberships.js

```diff
@@ -25,48 +25,48 @@ export default {
     destinationID: 'meeting1',
     destinationType: 'meeting',
     members: [
-      {personID: 'user1'},
-      {personID: 'user2'},
-      {personID: 'user3'},
+      {personID: 'user1', isInMeeting: false},
```

Comment:
Please make sure that you update the [interface/documentation](https://github.com/webex/component-adapter-interfaces/blob/master/src/MembershipsAdapter.js#L8) before making changes to the data model in other repos

Reply:
all right